<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StarCatalogueClass</class>
 <widget class="QMainWindow" name="StarCatalogueClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StarCatalogue</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>See only start that you study</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="RALabel">
            <property name="text">
             <string>RA</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="RALineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="decLabel">
            <property name="text">
             <string>Dec</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="decLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="diameterLabel">
            <property name="text">
             <string>Diameter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="diameterLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="updateButton">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="StarCatalogue.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked()</signal>
   <receiver>StarCatalogueClass</receiver>
   <slot>checkBoxChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>StarCatalogueClass</receiver>
   <slot>addStar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>StarCatalogueClass</receiver>
   <slot>updateStar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkBoxChanged()</slot>
  <slot>addStar()</slot>
  <slot>updateStar()</slot>
 </slots>
</ui>
